app/views/items/edit.html.erb

```diff
@@ -22,3 +22,19 @@
     <% end %>
   </div>
 </div>
+
+<div class="row">
+  <div class="col-md-10 col-md-offset-2 well">
+    <h4>Item History</h4>
+    <!--
+    We want to track:
+    * quantity change: 20 -> 25
+    * the reason for the change: donation, correction, purchase
+    * when the change occured: date
+    * and whodunit: user
+    -->
```

Comment:
This should be an ERB comment rather than an HTML comment. ERB comments won't show up in view source, so it's almost always better.

It would be done like this:
```erb
<%#
  We want to track:
  * quantity change: 20 -> 25
  * the reason for the change: donation, correction, purchase
  * when the change occured: date
  * and whodunit: user
%>
```

If it is a temporary comment that you intend to delete, then I guess it doesn't matter so much.